www/templates/default/html/js/manager.js

```diff
@@ -1,3 +1,7 @@
+require([frontend_url + 'templates/default/html/js/vendor/select2/js/select2.min.js', 'jquery'], function(select2) {
+	WDN.jQuery(".use-select2").select2();
```

Comment:
fixed in https://github.com/mfairchild365/UNL_UCBCN_System/commit/3a4cff35bfccf52bc7f29e088b21250db775f4e8